source/wp-content/themes/wporg-developer/scss/main.scss

```diff
@@ -804,18 +810,21 @@
 		}
 
 		.button-search {
-			background: transparent;
-			border: none;
+			background: #fff;
 			border-radius: 0;
 			box-shadow: none;
 			color: #32373c;
 			display: block;
-			height: 40px;
 			padding: 0.5rem 1rem;
-			position: absolute;
-			right: 0;
-			top: 0;
 			text-shadow: none;
+			border: 1px solid #ccc;
+			border-left: unset;
+			margin: 0;
+
+			&:active {
+				outline-style: auto;
+				outline-color: rgb(0, 95, 204);
```

Comment:
Where does this color come from? Maybe instead, try one of [the colors in this list](https://github.com/WordPress/wporg-developer/blob/d2060892add88a6be095d4a5dd3af525a2882793/source/wp-content/themes/wporg-developer/scss/settings/_colors.scss#L12) to keep the palette consistent. Like this:

```suggestion
				outline-color: get-color(blue-60);
```

Reply:
I got it from Chrome devTools directly.
![image](https://user-images.githubusercontent.com/18050944/172453508-c6e777f7-ccb1-4b32-84ad-5f7544c3da11.png)

> [the colors in this list](https://github.com/WordPress/wporg-developer/blob/d2060892add88a6be095d4a5dd3af525a2882793/source/wp-content/themes/wporg-developer/scss/settings/_colors.scss#L12) to keep the palette consistent

I like the idea, thanks for sharing it! 👍 However, after applying it, I found the color is a bit different in Chrome (it's hard to tell without zooming in or looking carefully into it though), and it's somehow the same in Firefox. I think I'll change it to use `get-color` here anyway as it seems to me it's just a slight difference.
![image](https://user-images.githubusercontent.com/18050944/172459082-ad11fb3a-c850-40a4-bdaa-24c0693890c3.png)